prometheus/metric_test.go

```diff
@@ -79,10 +79,14 @@ func TestWithExemplarsMetric(t *testing.T) {
 			}
 		}
 
-		infBucket := metric.GetHistogram().Bucket[len(metric.GetHistogram().Bucket)-1].GetUpperBound()
+		infBucket := metric.GetHistogram().Bucket[len(metric.GetHistogram().Bucket)-1]
 
-		if infBucket != math.Inf(1) {
-			t.Errorf("want %v, got %v", math.Inf(1), infBucket)
+		if upperBound := infBucket.GetUpperBound(); upperBound != math.Inf(1) {
+			t.Errorf("want %v, got %v", math.Inf(1), upperBound)
+		}
+
+		if cumulativeCount := infBucket.GetCumulativeCount(); cumulativeCount != 4711 {
+			t.Errorf("want %v, got %v", 4711, cumulativeCount)
```

Comment:
Can we stay consistent with code above?

```suggestion
		if want, got := infBucket.GetUpperBound(), math.Inf(1); want != got {
			t.Errorf("want %v, got %v", want, got)
		}

		if want, got := infBucket.GetCumulativeCount(), uint64(4711); want != got {
			t.Errorf("want %v, got %v", want, got)
```

Reply:
I pushed another commit to fix the assignment, as I didn't notice when I committed this that `want` and `got` were mixed up.